tests/runtest.cmd

```diff
@@ -177,6 +177,16 @@ xcopy /s "%__BinDir%" "%CORE_ROOT%"
 
 :SkipCoreRootSetup
 
+:: Pull down dependent packages needed for testing
+setlocal
+if defined __TestEnv call %__TestEnv%
+if defined COMPlus_GCStress set __Result=true
+endlocal & set __IsGCTest=%__Result%
```

Comment:
runtest.cmd uses __TestEnv to apply environment variables only to tests, not affecting build tools and test harness.